server/src/testE2E/java/com/flipkart/varadhi/E2EBase.java

```diff
@@ -74,6 +74,11 @@ static List<Project> getProjects(Response response) {
         });
     }
 
+    static List<String> getTopics(Response response) {
+        return response.readEntity(new GenericType<>() {
+        });
+    }
+
```

Comment:
Also need to take care of TODO below in the code for cleanup of project w.r.to topic deletion. TODO will still remain for other type of resources though.